backend/src/main/java/com/redhat/cloud/notifications/db/repositories/BehaviorGroupRepository.java

```diff
@@ -498,6 +499,26 @@ public void appendBehaviorGroupToEventType(final String orgId, final UUID behavi
         }
     }
 
+    @Transactional
+    public void appendActionToBehaviorGroup(final UUID behaviorGroupUuid, final UUID endpointUuid, final int position) {
+        final String appendBehaviorGroupQuery =
+            "INSERT INTO " +
```

Comment:
We should check (here or in an upper layer) that `endpointUuid` is owned by the current `orgId`.

Reply:
This check is akready done by a sub method called by an upper layer who will throw and exception, but it's not obious from a mantenance perspective. You are right, lets check it here to avoid confusion.